pkg/operation/botanist/component/coredns/coredns.go

```diff
@@ -90,6 +93,10 @@ type Values struct {
 	PodNetworkCIDR string
 	// NodeNetworkCIDR is the CIDR of the node network.
 	NodeNetworkCIDR *string
+	// UseClusterProportionalAutoscaling indicates whether cluster proportional autoscaling is enabled.
+	UseClusterProportionalAutoscaling bool
```

Comment:
Maybe just pass the autoscaling mode (instead of hard-coding one of many possible modes)?

Reply:
Adapted with next commit.